lib/actions/device.coffee

```diff
@@ -293,6 +308,71 @@ exports.hasDeviceUrl =
 			console.log(hasDeviceUrl)
 		.nodeify(done)
 
+exports.getApplicationInfo =
+	signature: 'device application info <uuid>'
```

Comment:
Minor point, but I _think_ this should be `app` everywhere, not `application`, to be consistent with existing comments (`resin apps`, `resin app restart`).